docs/document/docs/guides/format-setting-plugin.md

```diff
@@ -141,6 +144,24 @@ Place the JavaScript file (config.js) described below in the js folder.<br>
   // Write the process here
 })(kintone.$PLUGIN_ID);
 ```
+
+Place the CSS file (config.css) described below in the css folder.<br>
+
+***config.css***
+
+```css
+#kuc_cancel_button {
+  margin-right: 16px;
+}
+
+#kuc_dialog_footer {
+  text-align: center;
+}
+
+#kuc_dialog_cancel_button {
+  margin-right: 16px;
+}
+```
```

Comment:
I think it's better to place css description under all the js customization explanations following the actual customization steps.
How do you think?